port/ztpf/omrintrospect.c

```diff
@@ -1163,6 +1163,7 @@ omrintrospect_threads_startDo_with_signal(struct OMRPortLibrary *portLibrary, J9
 cleanup:
 	resume_all_preempted(data);
 	return NULL;
+	*/
```

Comment:
Is this function now missing a return?

Reply:
The return was located at the top of the function.